scripts/utils.ts

```diff
@@ -114,6 +115,77 @@ export async function findOrCreateSecurityGroup(ec2: EC2, groupName: string, des
   }
 }
 
+export async function createSecret(secretName: string, secretValue: string, region: string = "us-east-1"): Promise<void> {
```

Comment:
For all three of these methods, catch the error in the case of invalid permissions, and just console.warn that for now. I'm going to give the IAM user permission soon